scripts/deploy-new-vaults.ts

```diff
@@ -0,0 +1,108 @@
+import { readFileSync, writeFileSync } from 'node:fs'
+import { join, resolve } from 'node:path'
+import { parse } from 'valibot'
+import {
+  createPublicClient,
+  createWalletClient,
+  type Hex,
+  isAddressEqual,
+  zeroAddress,
+} from 'viem'
+import { privateKeyToAccount } from 'viem/accounts'
+import { berachain } from 'viem/chains'
+
+import {
+  type DefaultListPolVault,
+  DefaultListPolVaultsSchema,
+} from '@/schemas/pol-vaults-schema'
+
+import { INDENTATION_SPACES, INFRARED_ADDRESS } from './_/constants'
+import { infraredMainnetAbi } from './_/infrared-abi'
+import { transports } from './_/transports'
+
+const publicClient = createPublicClient({
+  chain: berachain,
+  transport: transports[berachain.id],
+})
+
+const walletClient = createWalletClient({
+  chain: berachain,
+  transport: transports[berachain.id],
+})
+const DEPLOYER_PRIVATE_KEY = process.env.DEPLOYER_PRIVATE_KEY as Hex
+
+const deployerAccount = privateKeyToAccount(`0x${DEPLOYER_PRIVATE_KEY}`)
+
+async function deployNewVaults(): Promise<Array<DefaultListPolVault>> {
+  const filePath = join(process.cwd(), 'src/pol-vaults/mainnet.json')
+  const fileContent = readFileSync(filePath, 'utf8')
+  const polVaults = parse(
+    DefaultListPolVaultsSchema,
+    JSON.parse(fileContent).vaults,
+  )
+
+  const polVaultsWithZeroAddress = polVaults.filter((vault) =>
+    isAddressEqual(vault.address, zeroAddress),
+  )
+
+  console.log(
+    `Found ${polVaultsWithZeroAddress.length} vaults with zeroAddress`,
+  )
+
+  if (polVaultsWithZeroAddress.length > 0) {
+    console.log('Pol vaults with zeroAddress:')
+    polVaultsWithZeroAddress.forEach((vault, index) => {
+      console.log(`${index + 1}. Slug: ${vault.slug}`)
+      console.log(`   Deposit Token: ${vault.depositTokenAddress}`)
+      console.log('')
+    })
```

Comment:
Remove?

Reply:
logs are nice to help debug. especially if it's for functionality that is basically one and done.